src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/OpenApiDocsTest.kt

```diff
@@ -63,4 +64,37 @@ class OpenApiDocsTest : IntegrationTestBase() {
       .expectStatus().isOk
       .expectBody().jsonPath("info.version").isEqualTo(DateTimeFormatter.ISO_DATE.format(LocalDate.now()))
   }
+
+  @Test
+  fun `the generated swagger for date times hasn't got the time zone`() {
+    webTestClient.get()
+      .uri("/v3/api-docs")
+      .accept(MediaType.APPLICATION_JSON)
+      .exchange()
+      .expectStatus().isOk
+      .expectBody()
+      .jsonPath("$.components.schemas.MappingDto.properties.whenCreated.example").isEqualTo("2021-07-05T10:35:17")
+      .jsonPath("$.components.schemas.MappingDto.properties.whenCreated.description")
+      .isEqualTo("Date time the mapping was created")
+      .jsonPath("$.components.schemas.MappingDto.properties.whenCreated.type").isEqualTo("string")
+      .jsonPath("$.components.schemas.MappingDto.properties.whenCreated.pattern")
+      .isEqualTo("""^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}${'$'}""")
+      .jsonPath("$.components.schemas.MappingDto.properties.whenCreated.format").doesNotExist()
+  }
+
+  @Test
+  fun `the security scheme is setup for bearer tokens`() {
+    val bearerJwts = JSONArray()
+    bearerJwts.addAll(listOf("read", "write"))
+    webTestClient.get()
+      .uri("/v3/api-docs")
+      .accept(MediaType.APPLICATION_JSON)
+      .exchange()
+      .expectStatus().isOk
+      .expectBody()
+      .jsonPath("$.components.securitySchemes.bearer-jwt")
+      .isEqualTo(mapOf("type" to "http", "scheme" to "bearer", "bearerFormat" to "JWT"))
+      .jsonPath("$.security[0].bearer-jwt")
+      .isEqualTo(bearerJwts)
```

Comment:
this is a bit naff but couldn't work out how to compare `list` other than using `JSONArray`